.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -154,8 +191,31 @@ jobs:
     container:
       image: ${{ inputs.image }}
       options: ${{ inputs.image_options }}
+    outputs:
+      run_conclusion: ${{ steps.run.conclusion }}
+      artifact_archive_name: ${{ steps.artifact_info.outputs.ARCHIVE_NAME }}
+      artifact_decompress_command: ${{ steps.artifact_info.outputs.DECOMPRESS }}
     env: ${{ fromJSON(inputs.env) }}
     steps:
+    - name: Deduce artifact archive params
+      if: inputs.upload_artifact == 'true'
+      id: artifact_info
+      run: |
+        NAME="${{inputs.artifact_archive_name}}"
+        if [ -z "$NAME" ]; then
+          NAME=e2e_binaries.tar.zst
+        fi
+        echo ARCHIVE_NAME="$NAME" >> $GITHUB_OUTPUT
+        if [ "${NAME}" != "${NAME%.tar.gz}" ]; then
```

Comment:
i wonder if this thing will work

https://github.com/ihiroky/extract-action

Reply:
@sarnex just wondering - is there any policy regarding regarding usage of non-github actions?

Reply:
I ended up hardcoding the type of compression we use, so I dont think this is necessary anymore